File: port/POO.java/Aula3.java
import java.util.Scanner;

public class Aula3 {
    public static void main(String [] args){
        int num1 = 0;
        int num2 = 1;
        int proxNum = 0;
        int n = 15;

        Scanner scanner = new Scanner(System.in);
        System.out.println("Os primeiros 15 termos da sequência de Fibonacci são :");
        System.out.println(num1 + num2);
        for (int i = 1; i < n; i++){
            proxNum = num1 + num2;
            num1 = num2;
            num2 = proxNum;
            System.out.println(proxNum);
        }
    }
}

File: port/POO.java/aula3.c
#include <stdio.h>

int main(){
    int num1 = 1, num2 = 1, proxNum = 0, n = 15;

    printf("Os primeiros %d termos da sequencia de Fibonacci eh: ", n);

    for (int i = 1; i <= n; i++){
        printf("%d ", num1);

        proxNum = num1 + num2;
        num1 = num2;
        num2 = proxNum;
    }

    printf("\n");

    return 0;
}